### ISSUE TITLE
training from scratch

### ISSUE THREAD

**Issue by kgeorge** [tensorlayer/hyperpose#199]
I have been doing training from scratch (not using any pre-initialized weights like vgg19.npz) on mscoco2017 dataset using vgg19 as the base model. I have gone past 5000 iterations, and the total_loss is not decreasing as a deep learning training usually do? Any idea why this may be happening? 
<img width="368" alt="opplus_total_loss" src="https://user-images.githubusercontent.com/947568/57571775-fb661180-73c6-11e9-89b1-5d1e64ff7932.png">

I also calculate the total loss for validation set, I compute that occasionally. That is also behaving oddly.

<img width="380" alt="opplus_val_los" src="https://user-images.githubusercontent.com/947568/57571792-3b2cf900-73c7-11e9-8b4c-cd94d2cb8fab.png">

**Comment by bela127** [comment]
Same problem here, when training from scratch, even after 600 000 steps.
(trained a little longer)

the loss is very jumpy... so no use full gradients can be calculated
also tested different learning rates, did not help.

my be you can tell us how you trained the hao28_experimental without the issues?

i will now try to do transfer learning with the hao28_experimental mentioned in:
#201
